docs/reference/rest-api/root.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[rest-api-root]]
+=== Root API
+++++
+<titleabbrev>Version API</titleabbrev>
+++++
+
+The Elasticsearch API's base url returns its basic build, 
+version, and cluster statistics. 
```

Comment:
I suggest "information" instead of "statistics" (these aren't numerical, nor stats).